ocaml/doc/basics.md

```diff
@@ -1,3 +1,7 @@
+---
```

Comment:
These headers are a particularity of XS docs. If there is an issue with having them here, I can remove them and have the pipeline insert them later (since it will have to replace the `@root@` placeholders in some files anyway).